website/src/categories/components/menu/usage.md

```diff
@@ -0,0 +1,148 @@
+
+### Reducing Visual Clutter
+Keeping a clean interface with repetitive elements (like cards or table rows), but additional actions are still necessary, a menu fixes this scenario while providing access to secondary actions. For example, sorting a list of elements or changing the view of the page.
+
+</section>
+
+
+<section>
+
+## When not to Use
+The following guidance describes when not to use the Menu component.
+
+### Primary Navigation
+Menus should not be used for primary navigation in applications because they are hidden until triggered by a specific action, making them less discoverable for users. Basic actions should always be visible and accessible to ensure users can efficiently navigate to essential features without additional steps. Relying on menus for primary navigation can also create inconsistency in accessing core functions by requiring users to learn multiple ways to perform basic tasks.
+
+</section>
+
+
+<section>
+
+## Anatomy
+
+|Item|Name| Description | Optional|
+|-|-|-|-|
+|1|Button Menu|This is the trigger button to open the menu. |no|
+|2|Menu Popover |This is the container for all the options of the menu, and appears close to the button menu. |no|
+|3|Option Items | This are the menu options that can be separete on "Option Button" that behave like a regular button, "Selection Option" that can behave like a checkbox or radio button, and "Sub-Menu Button" to open a nested menu. |no|
+|4|Group Heading |Title for the option groups to make eaiser to understand the function of the options. |yes|
+|5|Option Icons |You can display an icon in each option to enhance the readability. |yes|
+|6|Option Shortcuts |Adding shortcuts to the options to improve usability. |yes|
+|7|Group Heading |Title for the option groups to make eaiser to understand the function of the options. |yes|
+|8|Danger Option |Specific danger option of the menu. |yes|
+
+{.ds-table .ds-table-align-top}
+
+</section>
+
+
+<section>
+
+## Variants
+Menu comes in two versions, each suited for specific situations:
+
+  - **Button Option:** It triggers an immediate action or opens a submenu with further options. This variant is typically used for actions that have a direct and immediate effect, such as "Save," "Delete," or "Submit".
+
+  - **Single Select:** Allows users to choose only one option at a time. Clicking on an option deselects any previously selected items like a radio button. This variant is ideal for situations where users need to make mutually exclusive choices, such as change the sorting of a list or select the text editor in a text editor.
+
+  - **Multi Select:** This enables users to select multiple options simultaneously. Each option behave like a checkbox to toggle on and off the option, this can be applied to hide or show items from a list to filtering.
+
+</section>
+
+
+<section>
+
+## Options
+
+_ _ _ Currently Working on that in Figma _ _ _ 
```

Comment:
The documentation will be merged for now with `planned` status, so it will not be visible for people using the website, but we will need to remember about changing this part, when the component will be ready.

Reply:
@arecuenco-dsgn can you please take a look at this comment?